src/main/java/pilot/cards/pilot/A_Wall.java

```diff
@@ -0,0 +1,45 @@
+package pilot.cards.pilot;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.VulnerablePower;
+import pilot.PilotMod;
+import pilot.cards.CustomPilotModCard;
+import pilot.characters.Pilot;
+
+public class A_Wall extends CustomPilotModCard {
+    public static final String ID = PilotMod.makeID(A_Wall.class);
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Pilot.Enums.PILOT_CARD_COLOR;
+
+    private static final int COST = 2;
+    private static final int BLOCK = 13;
+    private static final int VULNERABLE = 3;
+    private static final int UPGRADE_PLUS_BLOCK = 3;
+
+    public A_Wall() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = BLOCK;
+        metaMagicNumber = baseMetaMagicNumber = VULNERABLE;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m){
+        addToBot(new GainBlockAction(p, p, BLOCK));
```

Comment:
This should be lower case 'block'. Lowercase block refers to the card's this.block variable, which is modified by things like upgrading and dexterity/momentum etc. Uppercase BLOCK refers to the static variable defined on line 21, and will always have the value 13.